pkgs/by-name/ni/nixos-rebuild-ng/src/nixos_rebuild/services.py

```diff
@@ -372,4 +373,5 @@ def write_version_suffix(build_flags: Args) -> None:
     nixpkgs_path = nix.find_file("nixpkgs", build_flags)
     rev = nix.get_nixpkgs_rev(nixpkgs_path)
     if nixpkgs_path and rev:
-        (nixpkgs_path / ".version-suffix").write_text(rev)
+        with contextlib.suppress(OSError):
```

Comment:
I would much prefer that we `logger.debug` here at least, so we have at least something in the `--debug` to understand what is happening here.